sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -2,62 +2,62 @@
 
 ## Introduction
 
-Azure SDK clients provide an interface to Azure services by translating library calls to REST requests. 
+Azure SDK clients provide an interface to Azure services by translating library calls to REST requests.
 
 The schematized body in the request or response, known as the message body, can be exposed in the Azure SDK client one of two ways:
 
 Most Azure SDK Clients expose methods that take ['model types'](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-model-types) as parameters, C# classes which map to the message body of the REST call. Those methods can be called here '**standard model methods**',
 
 However, some clients expose methods that mirror the message body directly. Those methods are called here '**protocol methods**', as they provide more direct access to the REST protocol used by the client library.
 
-### Pet's Example 
+### Pet's Example
 
-To compare the two approaches, imagine a service that stores information about pets, with a GetDog and SetDog pair of operations. 
+To compare the two approaches, imagine a service that stores information about pets, with a GetDog and SetDog pair of operations.
 
 Pets are represented in the message body as a JSON object:
 
 ```json
 {
-        "name": "Buddy",
-        "id": 2,
-        "color": "Brown"
+    "name": "Buddy",
+    "id": 2,
+    "color": "Brown"
 }
 ```
 
 An API using model types could be:
 
 ```csharp
```

Comment:
We should move this into a code snippet if possible.

Reply:
The whole document is based on a client which doesn't exists. Since this PR is for adding introduction to `RequestContext`, I try to control the scope of the change. It would be better to solve this in another issue. To be honest, topics in `ProtocolMethods.md` are covered in other documents, maybe we can even delete this.

Reply:
I've filed https://github.com/Azure/azure-sdk-for-net/issues/28315 to further track this issue.